src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: $card-width;
+  border: $card-border;
```

Comment:
The variable `$card-border` is used but not defined. Ensure it is defined in the `variables.scss` file or imported correctly.